sopel/db.py

```diff
@@ -124,33 +204,67 @@ def alias_nick(self, nick, alias):
         nick = Identifier(nick)
         alias = Identifier(alias)
         nick_id = self.get_nick_id(nick)
-        sql = 'INSERT INTO nicknames (nick_id, slug, canonical) VALUES (?, ?, ?)'
-        values = [nick_id, alias.lower(), alias]
+        session = self.ssession()
         try:
-            self.execute(sql, values)
-        except sqlite3.IntegrityError:
-            raise ValueError('Alias already exists.')
+            result = session.query(Nicknames) \
+                .filter(Nicknames.slug == alias.lower()) \
+                .filter(Nicknames.canonical == alias) \
```

Comment:
For this case, I prefer to do that:

```
session.query(Nicknames).filter(
    Nicknames.slug == alias.lower(),
    Nicknames.canonical == alias
).one_or_none()
```

which prevents the usage of `\`. I feel like it's more readable, but I didn't include that in my review, because I don't want to request a change I'm not sure is in-line with what @dgw think about it.

Reply:
@dgw thinks using `\` is kludgy and we should avoid it where possible. :smile_cat: